tests/models/opt/test_modeling_opt.py

```diff
@@ -331,7 +331,7 @@ def test_logits(self):
         assert torch.allclose(logits, logits_meta, atol=1e-4)
 
 
-@slow
+# @slow
 class OPTGenerationTest(unittest.TestCase):
     @property
     def prompts(self):
```

Comment:
Could you go into the test here and adapt it so that the prompt is:
```
"Today is a beautiful day and I want",
```
instead of 

```
"Today is a beautiful day and I want to"
```

Reply:
Done